# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/shell/jsshell.h
# Commit: e862899dca3f
# Full Hash: e862899dca3f252a8fe3c7be10d83d79dad328c5
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2018-05-16 18:32:17
# Regressor Bug: 1461751
# File Overlap Count: 2
# Description:
#   Bug 1461751 - Simplify module resolve hook to be a function pointer r=luke r=baku
# ==============================================================================

diff -r 1076e77e7b5c -r e862899dca3f js/src/shell/jsshell.h
--- a/js/src/shell/jsshell.h	Tue Apr 24 16:19:51 2018 +0200
+++ b/js/src/shell/jsshell.h	Wed May 16 11:59:09 2018 +0100
@@ -181,6 +181,7 @@
     js::Monitor offThreadMonitor;
     Vector<OffThreadJob*, 0, SystemAllocPolicy> offThreadJobs;
 
+    JS::PersistentRootedFunction moduleResolveHook;
 };
 
 extern ShellContext*